### DISCUSSION TITLE
difficulty with Japanese in a non-Japanese accent

### DISCUSSION THREAD

**Discussion by osamabinsaleem** [deepgram/community#714] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

 difficulty with Japanese in a non-Japanese accent or with words that should be written in roman characters.  The transcript has some problems when the client is speaking.  Additionally with some words that should be written in roman characters.

<img width="758" alt="Screenshot 2024-04-19 at 12 19 48 PM" src="https://github.com/deepgram/community/assets/37154675/927b6258-2823-47b6-889b-1444ccd7f738">
<img width="758" alt="Screenshot 2024-04-19 at 12 19 30 PM" src="https://github.com/deepgram/community/assets/37154675/0553dcd4-6f91-4632-96bf-8d30e0d536dd">

The above examples in the transcript do not have any meaning.

The transcript also enters grammar slightly differently than what is said.
In the below example Makie says 拝見させていただきたんですけど but the transcript says 拝見させていただいたのですが  

Functionally this is roughly the same in meaning but it's not what she said.  This seems to happen when someone is speaking quickly; the transcript will use slightly different grammar that has almost exactly the same meaning, or will drop out a word or phrase that isn't necessarily needed (like someone saying ummmm in English).

<img width="758" alt="Screenshot 2024-04-19 at 12 19 16 PM" src="https://github.com/deepgram/community/assets/37154675/8a1c046f-c8be-42a2-a838-a9e6c90ccc8d">

We use Nova-2 with automatic language detection feature enabled. Is there a way we can improve this?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
